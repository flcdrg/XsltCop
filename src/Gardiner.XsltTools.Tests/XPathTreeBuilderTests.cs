using System;
using System.Xml.Linq;

using ApprovalTests;
using ApprovalTests.Reporters;

using CodePlex.XPathParser;

using NUnit.Framework;

namespace Gardiner.XsltTools.Tests
{
    [TestFixture]
    public class XPathTreeBuilderTests
    {
        [Test]
        [UseReporter(typeof(BeyondCompareReporter))]
        public void AllFredElements()
        {
            var result = new XPathParser<XElement>().Parse("//fred", new XPathTreeBuilder());

            Approvals.VerifyXml(result.ToString());
        }
    }
}
