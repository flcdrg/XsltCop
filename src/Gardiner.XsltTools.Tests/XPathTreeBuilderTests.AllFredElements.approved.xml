<step>
  <Root nodeType="All" prefix="(null)" name="(null)" />
  <step>
    <DescendantOrSelf nodeType="All" prefix="(null)" name="(null)" />
    <Child nodeType="Element" prefix="" name="fred" />
  </step>
</step>